### PR TITLE
Update to 10.0.0

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

@conda-forge-admin please rerender

I'm pretty sure this will fail for now but I'll make the PR so it doesn't get forgotten

### CODE REVIEW

**Comment by timsnyder** [comment]
> License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

@chrisburr, I think if you change `about.license` to be `LicenseRef-MSVC_HEADERS`, the linter will be happy.

**Comment by timsnyder** [comment]
@conda-forge-admin, please lint

**Comment by timsnyder** [comment]
Ugh.  Now, the linter is complaining about the lower-case 'c' in 'BSD-3-clause'.  Needs to be 'BSD-3-Clause' and it will be clean.  Missed that one.

```
clang-win-activation on  patch-1 [!] via 🅒 base took 2s 
[I] ➜ git --no-pager diff
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index f4646f702..89f4cfe5b 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -59,7 +59,7 @@ outputs:
 
 about:
   home: https://github.com/conda-forge/clang-win-activation-feedstock
-  license: BSD-3-clause
+  license: BSD-3-Clause
   license_file: LICENSE.txt
   summary: 'Cross compiler for windows with MSVC ABI compatbility'
   description: |

clang-win-activation on  patch-1 [!] via 🅒 base 
[I] ➜ conda smithy recipe-lint recipe
recipe is in fine form
```

**Comment by timsnyder** [comment]
@conda-forge-admin, please restart ci

**Comment by timsnyder** [comment]
oops, I guess we still need compiler-rt=10.0.0 before this will be able to run.

**Comment by chrisburr** [comment]
Yup, I've retriggered it in: https://github.com/conda-forge/compiler-rt-feedstock/pull/19

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender

**Comment by tdegeus** [comment]
Thanks @chrisburr !!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,9 +32,8 @@ outputs:
       - unzip
       - curl
     about:
-      license: Proprietary
+      license: LicenseRef-MSVC_HEADERS
       license_file: LICENSE_MSVC_HEADERS.txt
-      license_family: Proprietary
```
Comment (isuruf) [inline recipe/meta.yaml:38]: Can you keep this here?

**Review by isuruf — APPROVED** [review]

**Comment by chrisburr** [comment]
I'll make a follow-up PR for 11 once this is merged

**Pull request merged**